packages/manager/modules/sms/src/sms/compose/telecom-sms-sms-compose.component.js

```diff
@@ -0,0 +1,14 @@
+import controller from './telecom-sms-sms-compose.controller';
+import template from './telecom-sms-sms-compose.html';
+
+export const component = {
+  controller,
+  template,
+};
+
+export const name = 'smsCompose';
```

Comment:
Is there any benefit to declare your component name in a const? Because it's only used once.

Reply:
I like having the component being self sufficient, if it contains its name, it can be reused without making spelling mistakes :)

in other words, its not used right now but it could be used one day or the other !

Reply:
I've just realized it's actually used twice:
- https://github.com/ovh-ux/manager/pull/389/files/2f942e914dbe05f6d1718490701826fc56343ce3#diff-b1787e6274e14fc4c3f8d19f14cdd1eeR24
- https://github.com/ovh-ux/manager/pull/389/files/2f942e914dbe05f6d1718490701826fc56343ce3#diff-93bb09ca0a92ca252bcb684a550e0ef5R10